src/modules/cpu/host_color_model_conversions.hpp

```diff
@@ -1553,7 +1553,7 @@ RppStatus hueRGB_processBuffer_host(T* srcPtr, RppiSize srcSize, T* dstPtr,
         dstPtrTempG = dstPtr + (imageDim);
         dstPtrTempB = dstPtr + (2 * imageDim);
 
-        Rpp64u alignedLength = bufferLength & ~3;
+        Rpp64u alignedLength = (bufferLength / 4) * 4;
```

Comment:
I think (bufferLength & ~3) will give the same result without division. Why it has been changed?

Reply:
Yes Rajy,
In past we were having some wrong results and suspected this would have given it and found out the bug was somewhere else.
Thank you.